connection.js

```diff
@@ -449,9 +440,9 @@ class Connection {
                 t.initial_time < config.minmaintime  || // fischer
                 t.max_time     < config.minmaintime  || // fischer
                 t.main_time    < config.minmaintime) {  // others
-                    let botSecondsTime = timespanToDisplayString(config.minmaintime);
-                    conn_log(user.username + " wanted main time below minmaintime " + botSecondsTime);
-                    return { reject: true, msg: "Minimum main time is " + botSecondsTime + ", please increase main time \n - If you use canadian byo-yomi, set the average time per stone, for example 5 minutes/25 stones needs to be set up as 180 seconds (5 minutes) divided by 25 stones , which equals 7.2 seconds per stone. " };
+                    let humanReadableTimeString = timespanToDisplayString(config.minmaintime);
+                    conn_log(user.username + " wanted main time below minmaintime " + humanReadableTimeString);
+                    return { reject: true, msg: "Minimum main time is " + humanReadableTimeString + ", please increase main time \n - If you use canadian byo-yomi, set the average time per stone, for example 5 minutes/25 stones needs to be set up as 180 seconds (5 minutes) divided by 25 stones , which equals 7.2 seconds per stone. " };
```

Comment:
Remove space before comma

Reply:
oh, its ok, this one will get replaced in the UHMAEAT anyways